### DISCUSSION TITLE
[논의] '책콩' 앱 캐릭터 이름에 대한 건

### DISCUSSION THREAD

**Discussion by DragonTiger92** [FRONTENDSCHOOL8/Book-Kong#90] [General]
저희 캐릭터 이름이 '콩시생'인가요??? 일단 프로젝트 테이블에는 그렇게 적혀있긴 한데...... 캐릭터 이름을 각자 다르게 부르는 것 같아 헷갈리네요. 앞으로 프로젝트를 소개하려면 용어 통일을 해야 모든 문서에 용어가 고르게 적힐 수 있을 것 같은데....... 혹시 다들 '콩시생'으로 알고 계신 건가요??????

**Comment by Samuel-0930** [comment]
넵 콩시생으로 생각하고 있습니다~

**Comment by DragonTiger92** [comment]
'콩시생'으로 앱 내 캐릭터 이름을 통일하여 인식하고 있는 것으로 판단 됨. 따라서 본 discussion을 종료함.
